core/src/main/java/com/yahoo/tagchowder/Schema.java

```diff
@@ -52,14 +52,35 @@
     private String theURI = "";
     private String thePrefix = "";
     private ElementType theRoot = null;
-    private boolean useIntern = true;
 
     /**
-     * Method to set useIntern.
-     * @param useIntern whether to use string intern or not
+     * Get the context of the parser.
+     * @return parserContext the context of parser
      */
-    protected void setUseIntern(final boolean useIntern) {
-        this.useIntern = useIntern;
+    public ParserContext getParserContext() {
+        return parserContext;
+    }
+
+    /**
+     * Set the parser context.
+     * @param parserContext the context of the parser
+     */
+    public void setParserContext(final ParserContext parserContext) {
```

Comment:
private

Reply:
made protected. This method is needed in Parser class to set it on the schema object.